components/string.rst

```diff
@@ -125,7 +125,7 @@ There are also some specialized constructors::
     $foo = ByteString::fromRandom(12);
 
     // CodePointString and UnicodeString can create a string from code points
-    $foo = UnicodeString::fromCodePoints(0x928, 0x92E, 0x938, 0x94D, 0x924, 0x947);
+    $foo = (string) UnicodeString::fromCodePoints(0x928, 0x92E, 0x938, 0x94D, 0x924, 0x947);
     // $foo = 'नमस्ते'
```

Comment:
hat about updating update this to `$foo = new UnicodeString('नमस्ते');` instead?

Reply:
Yes that would probably more consistent.

Reply:
But it's still a bit different from the style of the other comments, though :thinking:

Reply:
Maybe: `// $foo = UnicodeString('नमस्ते')` ?

Reply:
there is no function named UnicodeString...

Reply:
That's right... I just tried to make it a bit consistent with those other examples above:
```php
$contents = ByteString::wrap(['hello', 'world']);        // $contents = ByteString[]
$contents = UnicodeString::wrap(['I', '❤️', 'Symfony']); // $contents = UnicodeString[]
```
```php
$contents = UnicodeString::unwrap([
    new UnicodeString('hello'), new UnicodeString('world'),
]); // $contents = ['hello', 'world']
```
etc.

It's a bit tricky, though...

Reply:
Thanks Jacob! At the end, while merging this we did what Nicolas suggested and tweaked it as follows:

```
$foo = UnicodeString::fromCodePoints(0x928, 0x92E, 0x938, 0x94D, 0x924, 0x947);
// equivalent to: $foo = new UnicodeString('नमस्ते');
```

Reply:
Nice, thx! :)